src/main/java/ch/njol/skript/lang/Expression.java

```diff
@@ -103,9 +105,9 @@ default Optional<T> getOptionalSingle(Event e) {
 	 * Gets a non-null stream of this expression's values.
 	 *
 	 * @param e The event
-	 * @return A non-null stream of this expression's values
+	 * @return A non-null stream of this expression's non-null values
 	 */
-	default public Stream<? extends T> stream(final Event e) {
+	default public Stream<@NonNull ? extends  T> stream(final Event e) {
 		Iterator<? extends T> iter = iterator(e);
```

Comment:
```suggestion
	default public Stream<@NonNull ? extends  T> stream(Event event) {
		Iterator<? extends T> iter = iterator(event);
```
Mirroring Lime's comment as a suggestion type comment